tfx/examples/cifar10/README.md

```diff
@@ -0,0 +1,83 @@
+# CIFAR-10 Transfer Learning and MLKit integration Example
+This example illustrates how to use Transfer Learning for image classification with TFX, and use trained model to do object detection with [MLKit](https://developers.google.com/ml-kit)
+
+## Instruction
+
+Create a Python 3 virtual environment for this example and activate the
+`virtualenv`:
+
+```
+virtualenv -p python3.5 cifar10
+source ./cifar10/bin/activate
+```
+
+Next, install the dependencies required by the CIFAR-10 example (appropriate
+version of TF2 will be installed automatically).
+
+```
+pip install tfx
+```
+
+Then, clone the tfx repo and copy cifar10/ folder to home directory:
+
+```
+git clone https://github.com/tensorflow/tfx ~/tfx-source && pushd ~/tfx-source
+cp -r ~/tfx-source/tfx/examples/cifar10 ~/
+```
+### Dataset
+There is a subset of CIFAR10 (100 images) available in the data folder. To prepare the whole dataset, first create a script and run the following Python code:
+```
+import tensorflow_datasets as tfds
+ds = tfds.load('cifar10', data_dir='./cifar10/data/',split=['train', 'test'])
+```
+Then, create sub-folders for different dataset splits and move different splits to corresponding folders.
+```
+cd cifar10/data
+mkdir train_whole
+mkdir test_whole
+mv ciar10/3.0.2/cifar10-train.tfrecord-00000-of-00001 train_whole/
+mv ciar10/3.0.2/cifar10-test.tfrecord-00000-of-00001 test_whole/
+```
+### Train the model
+Execute the pipeline python file :
+```
+python ~/cifar10/cifar_pipeline_native_keras.py
+```
+The trained model can be found at `~/cifar10/serving_model_lite`
```

Comment:
I thought renaming it would make it more informative. By default the name is "tflite". I can remove the renaming part if you thing it's unnecessary.

Reply:
see below, given the assumptions of your metadata_writer, you must currently rename it. I agree it is more informative, but from an example / documentation standpoint it may be simpler to just go with tflite.

Reply:
sure sounds good